example/generated-src/objc/TXSItemList.h

```diff
@@ -7,6 +7,8 @@
 - (nonnull instancetype)initWithItems:(nonnull NSArray<NSString *> *)items;
 + (nonnull instancetype)itemListWithItems:(nonnull NSArray<NSString *> *)items;
 
+- (nonnull NSDictionary *) toDict;
```

Comment:
please suggest an alternative name that in your opinion follows the convention.
(in favor of saving time and effort, next time, please make a suggestion next to the original comment :) )